### PR TITLE
Homework 3

### PR DESCRIPTION
**Pull request by arianGorani**
PLS Review Homework 3

### CODE REVIEW

File: src/test/java/Selenium/Homework/Homework3.java [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        driver.manage().window().maximize();
+
+        try {
+            Thread.sleep(3000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+
+
+        String feelsLike = driver.findElement(By.className("feels-like-text")).getText().substring(0,2);
```
Comment (deepaktechnosoft) [inline src/test/java/Selenium/Homework/Homework3.java:38]: what if the temp is in single digit (eg: 1˚)?

File: src/test/java/Selenium/Homework/Homework3.java [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+        String lowTemp = driver.findElement(By.className("low-temp-text")).getText().substring(0,2);
```
Comment (deepaktechnosoft) [inline src/test/java/Selenium/Homework/Homework3.java:40]: same as above

File: src/test/java/Selenium/Homework/Homework3.java [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+        String highTemp = driver.findElement(By.className("high-temp-text")).getText().substring(0,2);
```
Comment (deepaktechnosoft) [inline src/test/java/Selenium/Homework/Homework3.java:42]: same as above

File: src/test/java/Selenium/Homework/Homework3.java [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+
+        int highTempInt = Integer.valueOf(highTemp);
+
+        boolean isFeelsLikeTempBtw = false;
+
+        if (feelsLikeInt >= lowTempInt && feelsLikeInt <= highTempInt){
+            isFeelsLikeTempBtw = true;
+        }
+
+        Assert.assertTrue(isFeelsLikeTempBtw,"FeelsLikeTemp is not between than low and high temperature");
```
Comment (deepaktechnosoft) [inline src/test/java/Selenium/Homework/Homework3.java:57]: Instead 
Assert.assertTrue(isFeelsLikeTempBtw,"FeelsLikeTemp is not between than low and high temperature");

you can also write, (Then don't need if-bloack and isFeelsLikeTempBtw-variable)
Assert.assertTrue((feelsLikeInt >= lowTempInt && feelsLikeInt <= highTempInt), "FeelsLikeTemp is not between than low and high temperature");

File: src/test/java/Selenium/Homework/Homework3.java [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+
+         driver.manage().window().maximize();
+
+         try {
+             Thread.sleep(3000);
+         } catch (InterruptedException e) {
+             e.printStackTrace();
+         }
+
+         String currentTempInF = driver.findElement(By.xpath("//span[@class='summary swap']")).getText().substring(0,2);
```
Comment (deepaktechnosoft) [inline src/test/java/Selenium/Homework/Homework3.java:92]: what if the temp is in single digit (eg: 1˚)?

File: src/test/java/Selenium/Homework/Homework3.java [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+
+         driver.findElement(By.xpath("//li[@class='last']")).click();
+
+         try {
+             Thread.sleep(3000);
+         } catch (InterruptedException e) {
+             e.printStackTrace();
+         }
+
+         String currentTempInC = driver.findElement(By.xpath("//span[@class='summary swap']")).getText().substring(0,2);
```
Comment (deepaktechnosoft) [inline src/test/java/Selenium/Homework/Homework3.java:114]: what if the temp is in single digit (eg: 1˚)?

File: src/test/java/Selenium/Homework/Homework3.java [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+
+
+         int faTemp = currentTempF;
+         int celTemp = (faTemp - 32) * 5/9;
+
+         System.out.println(faTemp + " °F" + "  -->  " + celTemp + " °C" );
+
+         boolean tempIsConverted = false;
+
+        if (currentTempC == celTemp
```
Comment (deepaktechnosoft) [inline src/test/java/Selenium/Homework/Homework3.java:130]: I don't think, this logic is correct

**Comment by arianGorani** [comment]
Hi Deepak I just modified my code and now they are working good, please can you review them and Marge it  on master branch :)

**Review by deepaktechnosoft — APPROVED** [review]

**Comment by deepaktechnosoft** [comment]
Merged

**Pull request merged**